debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCClassArrayClassSlotIterator.java

```diff
@@ -30,6 +30,7 @@
 import com.ibm.j9ddr.vm29.pointer.generated.J9ClassPointer;
 
 import static com.ibm.j9ddr.vm29.structure.J9Consts.*;
```

Comment:
I believe this import becomes unused: please remove it and similar imports elsewhere.